### PR TITLE
Hello world files

### PR DESCRIPTION
**Pull request by KDlamini**
Added 4 files;
-HTML file
-CSS file
-test.md file
-.gitignore
as well as updated README to be more descriptive.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:1]: - Kindly fix your Linter issues.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @KDlamini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (KDlamini) [reply .github/workflows/linters.yml:1]: Hi Carlos,

I think have fixed all the  Linter issues, but I'm not sure about how to request a re-review.
I have clicked the re-review button next to your name under reviewers but It doesn't seem to do anything.

**Review by dasileker — COMMENTED** [review]
### **Hi @KDlamini** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, you closed the PR and that's why I Invalidate the Project. please open a new PR and add All the necessary file and submit for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by KDlamini** [comment]
I have merged my Pull Request which closed it. What can I do now to proceed on the Dashboard?

**Pull request merged**